webscalesqlclient/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
 ExternalProject_Add(
   webscalesqlclient
+  URL file://${CMAKE_CURRENT_SOURCE_DIR}/webscalesql-5.6
```

Comment:
Yeah, I saw the error you posted earlier. Why is this just now coming up? Is it a new change in CMake? Why did they make the change? Is doing this directory the right URL? It seems to be shutting up cmake for the sake of shutting up cmake. What does the directory mean, and will specifying it this way hurt anything?

I think this is probably fine. I just want to try to understand it first.